app/controller/panel/TimeSlider.js

```diff
@@ -89,16 +97,34 @@ Ext.define('CpsiMapview.controller.panel.TimeSlider', {
      *
      * @param {Number[]} values The slider values
      * @param {Boolean} isRange Is range query or not
+     * @param {String} timeIncrementUnit The unit of time increment (year / month)
+     * @param {String} dateFormat The date format as listed in ExtJS documentation
+     * https://docs.sencha.com/extjs/6.2.1/classic/Ext.Date.html
      *
      * @returns {String} The formatted time string
      */
-    formatWmsDateString: function (values, isRange) {
+    formatWmsDateString: function (values, isRange, timeIncrementUnit, dateFormat) {
```

Comment:
The name is now a bit misleading since you are also able to format dates for WFS filters or am I getting this wrong?

Reply:
renamed to `formatDateString` in d4bdac8